Src/App/Router.php

```diff
@@ -1,31 +1,19 @@
 <?php
 namespace Aot\App;
 
-use Aot\Controller\PostController;
+use Aot\Controller\ApiController;
 
 class Router
 {
-    private $pc_instance;
-
-
-    public function __construct()
-    {
-        $this->pc_instance = new PostController();
-    }
 
     /**
-     * select the method
-     * @param  array $datas
-     * @return array
+     * lead to the controller
+     * @param $parameters
      */
-    public function method($datas = [])
+    public function route($parameters)
     {
-        if ($datas['method'] == 'show' && !array_key_exists('id', $datas)){
-            return $this->pc_instance->index($datas);
-        } elseif ($datas['method'] == 'show') {
-            return $this->pc_instance->show($datas);
-        } else {
-            throw new \Exception("not found");
-        }
+        $controller = new ApiController();
+        return $controller->sendToService($parameters);
```

Comment:
Parameters come from HTTP $_GET, and it is not secure, you may escape them in the router.
Also I'm not sure it's useful to forward `controller` and/or `method` parameters to controller

Reply:
the method `sendToService` should comes from `method` parameter, to be dynamic